Index: src/com/company/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/company/Main.java	(date 1541678898000)
+++ src/com/company/Main.java	(date 1541679300940)
@@ -3,6 +3,8 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        System.out.println("dod");
+        System.out.println("dod");
+
     }
 }
